### ISSUE TITLE
hello dear all.Could not get content for /account_parent/static/src/scss/coa_hierarchy.scss defined in bundle 'web.assets_common'.

### ISSUE THREAD

**Issue by canst** [odoo/odoo#82861]
Impacted versions:

Steps to reproduce:

Current behavior:

Expected behavior:

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by hameedullah** [comment]
Please use Odoo forum (https://www.odoo.com/forum/help-1) for help related queries. github issues should be used to report the issues with odoo.

**Comment by omalbastin** [comment]
Hi this is related to the third party app account_parent you are using. Please contact them through odoo apps.

**Comment by mart-e** [comment]
Indeed, the github issues are to report software bugs on standard odoo modules
